test/kafka-e2e/kafka_test.go

```diff
@@ -13,7 +13,8 @@ import (
 )
 
 //go:generate kubectl -n numaflow-system delete statefulset zookeeper kafka-broker --ignore-not-found=true
-//go:generate kubectl apply  -k ../../config/apps/kafka -n numaflow-system
+//go:generate kubectl apply -k ../../config/apps/kafka -n numaflow-system
+// Wait for zookeeper to come up
```

Comment:
is there way we can verify that zookeeper is up?

Reply:
It's difficult. Technically, the order should be zookeeper comes up, and then broker comes up. Needs more effort to do that, simply wait for a while to make it work.